msg/versioned/TrajectorySetpoint.msg

```diff
@@ -12,6 +12,8 @@ float32[3] position # in meters
 float32[3] velocity # in meters/second
 float32[3] acceleration # in meters/second^2
 float32[3] jerk # in meters/second^3 (for logging only)
+float32[4] q_d # attitude quaternion
+float32[3] angular_velocity # in radians/second
```

Comment:
The definition of the trajectory_setpoints message is based on the property that multirotors and fixed-wings(under specific condition) is differentially flat. 

- Adding attitude fields in this message, basically breaks this as attitude or angular velocity should be defined through the flat outputs (acceleration or jerk). 
- As most aerial vehicles do not use q_d or angular_velocity, this would just result in decreasing the performance of sending and receiving this message.

I guess you are looking at more fully actuated vehicles, that are not differentially flat. I would suggest that this would probably belong to a separate message that can be useful for your application.

Reply:
In that case I'll create a new message (FullyActuatedTrajectorySetpoint.msg or 6DoFTrajectorySetpoint.msg) and use that, thanks!